spec/integration/client_spec.rb

```diff
@@ -57,7 +60,29 @@ def push_test_images
     client = Portus::RegistryClient.new(registry_hostname)
     manifest = client.manifest("registre", "2.3.1")
 
-    expect(manifest["name"]).to eq "registre"
-    expect(manifest["tag"]).to eq "2.3"
+    expect(manifest[0]).to_not be_empty
+    expect(manifest[1]).to_not be_empty
+    expect(manifest[2]["schemaVersion"]).to eq 2
+  end
+
+  it "supports deleting manifests" do
+    push_test_images
+
+    client = Portus::RegistryClient.new(registry_hostname)
+
+    # Provided digest did not match uploaded content
+    expect do
+      client.delete("registry", "itsdangeroustogoalonetakethis", "manifests")
+    end.to raise_error(RuntimeError)
+
+    _, dig, = client.manifest("registre", "2.3.1")
```

Comment:
Maybe "registre" and "2.3.1" should be placed inside of variables (maybe defined via `let`).